packages/label/src/LabelBase.tsx

```diff
@@ -0,0 +1,151 @@
+import {
+  critical20,
+  informative20,
+  success20,
+  warning20,
+} from '@littlespoon/theme/lib/colors/alert'
+import { brand60 } from '@littlespoon/theme/lib/colors/primary'
+import { gold60, limeGreen60, peach60, pink60 } from '@littlespoon/theme/lib/colors/secondary'
+import { shadeBlack } from '@littlespoon/theme/lib/colors/token'
+import { family, weight } from '@littlespoon/theme/lib/fonts/primary'
+import { rem } from '@littlespoon/theme/lib/utils'
+import styled from 'styled-components'
+
+import type { LabelProps } from './Label'
+
+export const LabelBase = styled.label<LabelProps<'label'>>`
```

Comment:
Unfortunately, this was never explicit in the requirements, but this Label component is not meant to represent a form Label. It's just some sort of tag/label to display information.

I don't think we should use the `label` tag, but a `span`.

https://stackoverflow.com/questions/33214093/is-it-good-practice-to-use-label-for-non-input-non-interactive-elements

We'll be using this component to replace this FE component https://github.com/little-spoon-dev/little-spoon-frontend/blob/main/app/components/modules/ProductLabelV2/index.js

It's in use here littlespoon.com/products/plates/78/mac-and-three-cheese

Let me know what you think. @ryanjdiercks @remarkablemark

Reply:
Gotcha if that's the case, then I'm ok with `span`

I do want to note that I did see an answer that says it's ok to use `label` without `input` or `form`: https://stackoverflow.com/questions/54107105/is-using-a-label-without-any-associated-form-element-acceptable

> You may use a label without an associated form element. The spec notes that a label can be associated with a form element, or contain the element within itself, but does not specify that either is required.
>
> A label is expected to be used where phrasing content is expected.

Reply:
Solved it